src/components/UpdateWhitelistUsersButton/AddressValidationModal/styles.less

```diff
@@ -0,0 +1,41 @@
+.whitelist-users-address-validation {
+  .info-wrapper {
+    .info-row {
+      padding: 4px 0;
+    }
+  }
+  .@{app-prefix}-table-body {
+    padding-top: 20px;
+  }
+  .@{app-prefix}-hash-address span {
+    color: #070a26;
```

Comment:
.

Reply:
It will be improved in the next version.